src/lib/pure_pursuit/PurePursuit.hpp

```diff
@@ -50,6 +53,10 @@ using namespace matrix;
  * Pure pursuit is a path following algorithm that uses the intersection between the path and
  * a circle (the radius of which is referred to as lookahead distance) around the vehicle as
  * the target point for the vehicle.
+ * The lookahead distance is defined as v * k.
+ * 	v: Vehicle ground speed [m/s]
+ * 	k: Tuning parameter (Implemented as PP_LOOKAHD_GAIN)
```

Comment:
It's often not a good idea to mention the name of a parameter or explain something that is done elsewhere as we have no control over it right here. For the parameter here, we will just hope that the next person changing the parameter name won't forget to change that comment as well.

Reply:
changed in acd8e20